src/main/resources/import.sql

```diff
@@ -1,9 +1,9 @@
 
 
-INSERT INTO Product (category,name,price) VALUES ('equipment','filter',12);
-INSERT INTO Product (category,name,price) VALUES ('equipment','heater',15);
-INSERT INTO Product (category,name,price) VALUES ('equipment','Termometer',8);
-INSERT INTO Product (category,name,price) VALUES ('equipment','Tank',200);
+--INSERT INTO Product (category,name,price) VALUES ('equipment','filter',12);
```

Comment:
Good point, I think we can remove sample products imports if we're using starter class for testing purposes.